packages/ember-metal/lib/property_get.js

```diff
@@ -51,24 +50,17 @@ export let UNHANDLED_GET = symbol('UNHANDLED_GET');
   @public
 */
 export function get(obj, keyName) {
+  Ember.assert(`Get must be called with two arguments; an object and a property key`, arguments.length === 2);
+  Ember.assert(`Cannot call get with '${keyName}' on an undefined object.`, obj !== undefined && obj !== null);
+  Ember.assert(`Cannot call get with ${keyName} key.`, typeof keyName === 'string');
```

Comment:
This assertion could be clearer with something like `The key provided to get must be a string, you passed ${keyName}`